.travis.yml

```diff
@@ -48,8 +48,6 @@ matrix:
       env: COMPOSER_FLAGS="--prefer-lowest"
     - php: '7.2'
       env: SYMFONY=2.8.*
```

Comment:
Didn't we drop 2.8 - 3.3 on master?

Reply:
Yes, but until this gets merged, dev-kit wont trigger